SU-SWS/saa_victoria_subtheme
# Adapt 401  buttons cta color updates & footer buttons & CTA

# READY FOR REVIEW

# Summary
- Adjusted button hover colors, removed red buttons, and improved cta colors
- set link and button styles for footer as well (JIRA ADAPT-376)

# Review By (Date)
- 6/29

# Urgency
- Nice since it affects multiple components

# Steps to Test

1. pull down, clear caches
1. Check button styles for purple button, secondary button in cards
2. create top banner with image, check a link inside a paragraph (plain link), check an action link after a paragraph (with action link class), and the white button on red (without a purple border) with correct purple hover. see figma
3. check link in a card as well for correct cardinal red hover color and check it with an action link and icon
https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=192%3A384
4. Test that the footer links and login button match figma: https://www.figma.com/file/TXNBECAGcEkZ3vwc6xcZ3g/SAA-Design-System?node-id=897%3A3967
5. also add a sign up form into the footer and check that that button matches the styles also

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ADAPT-401
- JIRA ADAPT-376
